src/tests/cli/firewall-cmd.at

```diff
@@ -4,7 +4,15 @@ m4_ifdef([TESTING_FIREWALL_OFFLINE_CMD], [], [
 
 FWD_START_TEST([basic options])
     AT_KEYWORDS(panic reload)
+    
+    FWD_CHECK([], 2, [State: running
```

Comment:
The status information is going to `stdout`. The usage message is going to `stderr`. I think we have we have to send both to `stderr` otherwise the lines may be interleaved when executed interactively. This output is intended for the user and not meant to be piped to another program.

@todoleza, ping just in case you have input about this.